packages/select/src/components/multi-select/multiSelect2.tsx

```diff
@@ -289,6 +289,7 @@ export class MultiSelect2<T> extends AbstractPureComponent2<MultiSelect2Props<T>
                     ...popoverTargetProps,
                     ...targetProps,
                     "aria-expanded": isOpen,
+                    "aria-disabled": disabled,
```

Comment:
you need to swap these lines (yes, I know the lint rule is kind of annoying)

Reply:
lint rules are there for good reason!